ATO-245: ADD  PerformanceGenerator.C(AliGenPerformance.cxx)

miranov25 opened the pull request:
### Description
* Performance generator to be used for fast MC pre-production validation
* Make performance generator with flat 1/pt and flat pdg () input jets
  * Jet multiplicity smeared (Poisson) with mean value nJets
  * nJets  defaults 1 (pp), 2(pPb), 10 (PbPb)
    * can be overwritten by env variable PerformanceGenerator_nJets

#### Tests:
* Before pull request  PerformaceGenerator.C was tested for pp and PbPb pre-production for ALIROT-7070 (GSI)
  * see discussion in https://alice.its.cern.ch/jira/browse/ALIROOT-7077?focusedCommentId=206353&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-206353

fprino commented:
Hi @miweberSMI,

for me the generator prepared by Marian can be committed.
If you also agree we can merge it.

Cheers, Francesco

miweberSMI commented:
Dear @miranov25 

to me the commit looks good, except that the name of the macro has a small typo.
Perhaps you still could correct this?

Cheers,
Michael

miranov25 commented:
Hallo Michael.

* Thank you for spotting typo. I renamed the file.
* I also fixed a problem in AddTaskFilteredTree.C in AliPhsycis_SRC which you reported
  * see https://github.com/alisw/AliPhysics/pull/3213
  * your test should be fine now
    * in my previous test the macro did not enter to affected line as I always set parameters
 
Marian

miweberSMI commented:
Hi Marian,

I tested with latest master of AliRoot/AliPhysics/AliDPG + this pull request and everything runs smoothly. I will merge this commit.

Thanks and cheers,
Michael

fprino commented:
Hi Marian,

you can open a Jira ticket in AliRoot (there is no AliDPG project in Jira), using the type "new development".

Cheers, Francesco
